src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineInstance.java

```diff
@@ -80,7 +82,15 @@ protected void _terminate(TaskListener listener) throws IOException, Interrupted
 
 
     }
-    
+
+    /**
+     * Based on the instance configuration, whether to create snapshot for an instance with failed builds at deletion tiem.
```

Comment:
s/tiem/time
No need to supply the type in the \@return annotation, javadoc handles that for you.

Reply:
done